recipes/pkgconf/all/conanfile.py

```diff
@@ -114,6 +114,9 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "share"))
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
 
+    def package_id(self):
+        del self.info.settings.compiler
```

Comment:
This recipe also provides a library, which requires a compiler present in the settings + package id.
Unless no other recipe depends on the library, I'm fine with stopping providing it, or optionally providing it as an option.

Reply:
owww, I forgot that right. I'll add an option.

Reply:
@madebr good catch!